usr/share/job/templates/interactive-desktop/metadata.yaml

```diff
@@ -7,6 +7,7 @@ version: 0
 tags:
   - script:type=interactive
   - session:type=desktop
+  - session:host=node
```

Comment:
This isn't quite the right name.  It isn't really about the host as much as it is the ordering `flight desktop -> srun` vs `sbatch -> flight desktop`.  Perhaps `session:order=desktop:scheduler_alloc` / `session:order=scheduler_alloc:desktop`.